bindings/py/packaging/setup.py

```diff
@@ -232,6 +263,29 @@ def generateExtensions(platform, build_type):
   Note: for Windows it will force a X64 build.
   """
   cwd = os.getcwd()
+  scriptsDir = os.path.join(REPO_DIR, "build", "scripts")
+  try:
+    if not os.path.isdir(scriptsDir):
+      os.makedirs(scriptsDir)
+    os.chdir(scriptsDir)
+
+    # cmake ../..
+    configure(platform, build_type)
+
+    # build: make && make install
+    if platform != "windows": #TODO since cmake 3.12 "-j4" is directly supported (=crossplatform), for now -- passes other options to make
+      subprocess.check_call(["cmake", "--build", ".", "--target", "install", "--config", build_type, "--", "-j", "4"]) 
```

Comment:
we now call cmake/make with implicit `-j4`, this really speeds up builds using `python setup.py` for users and developers. 
- on CI the build time is 2x faster (down to 6-10min, from 20+)
- crossplatform approach since cmake 3.12, but I don't find it reasonable to bump dependency, it's too new yet.
- Windows slows down the green PRs now, I tried building with `/MT` instead of `/MD` (which I found is equivalent of using `make -j`, but then unittests complain to be compiled with /MD, so reverted for now. FYI @dkeeney